api-tests/src/steps/ipv-steps.ts

```diff
@@ -145,3 +147,21 @@ Then(
     }
   },
 );
+
+Then(
+  "my proven user details match",
+  async function (this: World): Promise<void> {
+    assert.ok(
+      isPageResponse(this.lastJourneyEngineResponse),
+      `got a ${describeResponse(this.lastJourneyEngineResponse)}`,
+    );
+    assert.equal("page-ipv-reuse", this.lastJourneyEngineResponse.page);
```

Comment:
Do we need this assertion? We actually use this endpoint on a couple of other pages now - particularly in the update details journey and repeat fraud check.